.github/workflows/test.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         if: steps.fast-tests.outcome == 'success'
```

Comment:
🤔 Are the selenium tests considered unit tests, or integration tests?

Reply:
I think it's user interface testing; it doesn't test the integration with the backend services so I wouldn't call it integration testing